assets/styl/_components/modal.twig

```diff
@@ -0,0 +1,8 @@
+<!-- We overide some css of `.modal` class to not make it overlay all the viewport -->
+<div class="modal" style="height:50vh; max-height:25rem; position:relative; overflow:hidden;" >
+    <div class="modal-overlay" data-a11y-dialog-hide="true"></div>
+    <div class="modal-window box" role="dialog" open="">
+        <button class="modal-closebtn btnIcon" data-a11y-dialog-hide="true"><svg height="12" width="12"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#symbol-cross"></use></svg></button>
```

Comment:
I would have put the button after `modal-content`, to be sure it displays above the content